templates/patroni.yml.j2

```
@@ -21,6 +21,9 @@ log:
 restapi:
   listen: '{{ self_ip }}:8008'
   connect_address: '{{ self_ip }}:8008'
+  authentication:
+    username: operator
+    password: {{ patroni_password }}
```

let's do it consistently to avoid confusion e.g. why operator_password doesn't work for the operator user in patroni
```suggestion
    username: patroni
    password: {{ patroni_password }}
```